src/TcpPolyfill.js

```diff
@@ -56,14 +56,45 @@ export function Socket(options) {
 
     ws.onmessage = event => {
       const data = event.data;
+
+      // Handle incoming message as base64 if ws.protocol is set to base64
+      // or if it's not set at all and we are in a problematic React Native
+      // environment per:
+      // https://tools.ietf.org/html/rfc6455#section-11.3.4
+      // https://tools.ietf.org/html/rfc3864
+
+      const base64SetByDevice = (
```

Comment:
Alright, thought about this some more. Renamed variable to `base64SetByWsClient`. I suppose we could go all out and just called it `base64SetByWsAfterMessageReceivedByServer` 😝 but this presumed since it wouldn't be set outside the context of the response message.

I still feel `base64SetByServer` isn't super clear because the server behaves consistently. Its `ws` that seems to have this quirk given its environment (i.e., RN).